.github/workflows/_reusable-verify-run-status.yaml

```
@@ -0,0 +1,47 @@
+# Prefix with "~" to sort last in Actions list
+name: ~Templates - Verify Run Status
+
+on:
+  workflow_call:
+    inputs:
+      check_run_name:
+        description: Name of the check run to verify
+        required: true
+        type: string
+      workflow_name:
+        description: Name of the workflow to verify
+        required: true
+        type: string
+
+permissions:
+  checks: read
+
+jobs:
+  check-run-status:
+    runs-on: ubuntu-24.04
+
+    steps:
+      # *** IMPORTANT ***
+      # For workflows that are triggered by the pull_request_target event, the workflow runs in the
+      # context of the base of the pull request.  You should make sure that you do not check out,
+      # build, or run untrusted code from the head of the pull request.
+      - uses: actions/checkout@v4
+        with:
+          # Only needs .github folder for automation, not the files in the PR (analyzed in a
+          # separate workflow).
+          #
+          # Uses the .github folder from the PR base branch (pull_request_target trigger),
+          # or the repo default branch (other triggers).
+          sparse-checkout: |
+            .github
```

```suggestion
      - uses: actions/checkout@v4
        with:
          sparse-checkout: |
            .github
```

This workflow doesn't (and shouldn't) trigger on pull_request_target, so this comment doesn't apply.  I think you might have copy/paste this from another workflow with a different trigger.

I think you just want to copy this:

https://github.com/Azure/azure-rest-api-specs/blob/9191854514911a10bfd71636fd2903e6be2d7edf/.github/workflows/_reusable-eng-tools-test.yaml#L45-L48